app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+    validates :body, presence: true
+    validates :user_id, presence: true, numericality: { greater_than: 0 }
```

Comment:
for foreign key, we don't need to validate. user_id and post_id is a foreign key. For this, we don't need any validation. It's always recommended to use `build` to the controller so that it will automatically build with associated tables foreign key data.